tidy3d/plugins/adjoint/web.py

```diff
@@ -65,7 +65,7 @@ def _task_name_adj(task_name: str) -> str:
 
 def tidy3d_run_fn(simulation: Simulation, task_name: str, **kwargs) -> SimulationData:
     """Run a regular :class:`.Simulation` after conversion from jax type."""
-    return web_run(simulation=simulation, task_name=task_name, **kwargs)
+    return web_run(stub=simulation, task_name=task_name, **kwargs)
```

Comment:
should this be `return web_run(simulation=simulation, task_name=task_name, **kwargs)`

Reply:
I think so